doc/apidocs/javadocs/allclasses-frame.html

```diff
@@ -1,3 +1,18 @@
+<!--
+
+    (c) Copyright 2016 Admios. The software in this package is published under the terms of the CPAL v1.0 license, a copy of which has been included with this distribution in the LICENSE.md file.
+
+-->
+<!--
+
+    Admiosss. The software in this package is published under the terms of the CPAL v1.0 license, a copy of which has been included with this distribution in the LICENSE.md file.
+
```

Comment:
This look to be repeated 3 times, I think the first line is the correct and the other are just not necessary.